checks/pull_requests.go

```diff
@@ -75,9 +75,10 @@ func PullRequests(c *checker.CheckRequest) checker.CheckResult {
 			totalWithPrs++
 			c.Dlogger.Debug("commit with PR: %s", commit.GetSHA())
 		} else {
-			c.Dlogger.Debug("!! found commit without PR: %s, committer: %s", commit.GetSHA(), commit.GetCommitter().GetLogin())
+			c.Dlogger.Debug("found commit without PR: %s, committer: %s", commit.GetSHA(), commit.GetCommitter().GetLogin())
```

Comment:
all failed messages try to follow by adding "!! " at start. if you want to change it everywhere, that is fine, but best to do in another PR. i slightly liked it, although slightly confusing as well since not standard convention.

Reply:
none of the messages follow this anymore. We've changed that when we moved to score-based results. Now we use an enum `Warn/Info/Debug` to infer *what* the message is about. This one caught my eye because it slipped thru the cracks :-)